artifacts/definitions/Windows/Remediation/Quarantine.yaml

```diff
@@ -0,0 +1,359 @@
+                            'action=' + Action
+                                ) AS CMD
+                    FROM scope()
+                    WHERE FilterAction AND Action =~ 'Permit|Block'
+                },
+                rule={
+                    SELECT 
+                        ('netsh','ipsec','static','add','rule',
+                            'name=' + PolicyName + '_' + Action,
+                            'policy=' + PolicyName,
+                            'filterlist=' + PolicyName + '_' + Action + 'FilterList',
+                            'filteraction=' + PolicyName + '_' + Action + 'Action'
+                                ) AS CMD
+                    FROM scope()
+                    WHERE 
+                        Rule AND Action =~ 'Permit|Block'
+                },
+                enable={
+                    SELECT 
+                        ('netsh','ipsec','static','set','policy',
+                            'name=' + PolicyName,
+                            'assign=y'
+                                ) AS CMD
+                    FROM scope()
+                    WHERE _value = 'EnablePolicy'
+                },
+                policy= {
+                    SELECT
+                        ('netsh','ipsec','static','add','filter',
+                            'filterlist=' + FilterList,
+                            'srcaddr=' + SrcAddr,
+                            'srcmask=' + SrcMask,
+                            'srcport=' + SrcPort,
+                            'dstaddr=' + DstAddr,
+                            'dstmask=' + DstMask,
+                            'dstport=' + DstPort,
+                            'protocol=' + Protocol,
+                            'mirrored=' + Mirrored,
+                            'description=' + Description
+                                ) AS CMD
+                    FROM scope() 
+                    WHERE 
+                        SrcAddr AND DstAddr              
+                })
+            
+      # delete old or unwanted policy 
+      - LET delete_policy = SELECT * FROM foreach(
+                row=['DeletePolicy'],
+                query={
+                    SELECT 
+                        timestamp(epoch=now()) as Time,
+                        PolicyName + ' IPSec policy removed.' AS Result
+                    FROM 
+                        execve(argv=cmdline[0].CMD,sep='\r\n')
+                    GROUP BY Stdout
+                })
+      
+      # first step is creating IPSec policy
+      - LET create_policy = SELECT * FROM foreach(
+                row=['CreatePolicy'],
+                query={
+                    SELECT 
+                        timestamp(epoch=now()) as Time,
+                        if(condition=Stdout, 
+                            then=Stdout, 
+                            else= if(condition=Stderr,
+                                then=Stderr,
+                                else= if(condition= ReturnCode=0, 
```

Comment:
dont we always want to look at the return value?

Reply:
In testing the ReturnCode could be 0 and still provide an error to Stdout or no Stdout on successful execution.
I dont recall seeing any Stderr but thought it would be prudent to add in too incase there was an edgecase I was missing. 

StdOut was only errors that I saw or some information about badly entered fields so I thought valuable to prioritise.

Reply:
I think an area of future improvement here could be research into edgecases on mis entered fields in the custom config.